### ISSUE TITLE
Transfer Coin Issue

### ISSUE THREAD

**Issue by deshario** [kulapio/libra-service#16]
Everything's working fine except transfer api it doesnot validate or anything always return status:200

I dont have enough coin to transfer but also it can send request to transfer api ... but actually transaction never happened

``````


 axios
        .post("https://libraservice2.kulap.io/transfer",{
            "fromAddress": source, // 0da045bec135aaaf280c16afd9671162f6c6809a9593b7b6e7be1175f7bbba06
            "mnemonic": "lorem ipsum",
            "toAddress": destination, // 7d7041e1c6635cb8bf55b1f064487553046af9dc2079813d2a5c741a9ad5a875
            "amount": amount // 20000000
        })
        .then(response => {
            if(response.status == 200){
                console.log('Transfer Success');
            }
            console.log('response ==> ',response)
        })
        .catch(err => {
            console.log(err);
            return null;
        });
``````
![Screen Shot 2019-07-09 at 4 31 58 PM](https://user-images.githubusercontent.com/11426727/60877142-6cab2f00-a267-11e9-8f28-07d248a49414.png)

**Comment by totiz** [comment]
Thank for reporting, yeah that's a bug, we need to fix it, If you've a chance please feel free to PR.

**Comment by totiz** [comment]
@deshario Fixed with 1ab2d2ec8e1f571dacba8ca6ee8664d446236cc5.
